test/Azure.Functions.Cli.Tests/PublishActionTests.cs

```diff
@@ -14,38 +14,26 @@ public class PublishActionTests
     {
         TestAzureHelperService _helperService = new TestAzureHelperService();
 
-        [Fact]
-        public async Task NetFrameworkVersion_DotnetIsolated_Linux_Consumption_AlreadyEmpty()
+        [Theory]
+        [InlineData(null)]
+        [InlineData("something")]
+        public async Task NetFrameworkVersion_DotnetIsolated_Linux_Consumption_Updated(string initialLinuxFxVersion)
         {
```

Comment:
please add tests for `dotnet-isolated|5.0`, `dotnet-isolated|6.0` and `dotnet-isolated|7.0`,